client/state/sites/sharing-buttons/reducer.js

```diff
@@ -1,124 +0,0 @@
-import { uniqBy } from 'lodash';
-import {
-	SHARING_BUTTONS_RECEIVE,
-	SHARING_BUTTONS_REQUEST,
-	SHARING_BUTTONS_REQUEST_FAILURE,
-	SHARING_BUTTONS_REQUEST_SUCCESS,
-	SHARING_BUTTONS_SAVE,
-	SHARING_BUTTONS_SAVE_FAILURE,
-	SHARING_BUTTONS_SAVE_SUCCESS,
-	SHARING_BUTTONS_UPDATE,
```

Comment:
You also want to remove these action types from `state/action-types`.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/56579/commits/aa360906b9b8d9f7662d12781b1abfb743286068